### ISSUE TITLE
bug: universal patches are always shown

### ISSUE THREAD

**Issue by Ushie** [ReVanced/revanced-manager#1080]
### Bug description

Universal patches are shown in the patches selector screen regardless of whether or not "Experimental universal patches support" is enabled in ReVanced Manager settings

### Solution

Hide universal patches in patches selector if the setting is not toggled on

### Acknowledgments

- [X] I have searched the existing issues; this is new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I properly filled out all of the requested information in this issue.
- [X] The issue is solely related to ReVanced Manager and not caused by patches.

**Comment by TheAabedKhan** [comment]
Are they just shown or selectable as well?? If they are not selectable then that's not an issue because user needs to see that there are other patches as well which are, although, disabled.

However, If they are causing disturbance in patches selection, I can add a chip that can hide/show unsupported patches, but they will be able to use them only after enabling the toggle in the settings page.

**Comment by Ushie** [comment]
They are selectable, that's where the issue lies, people with universal patches not toggled on might not realise that these patches aren't specific to that package, as it looks like any other, it might be good to separate them too or add an indicator

**Comment by TheAabedKhan** [comment]
![Screenshot_20230804-164942_Pixel Launcher Extended.png](https://github.com/ReVanced/revanced-manager/assets/39409020/bb3a1832-4bad-40bf-8b42-f4621cfe4823)

I am unable to reproduce your issue. Can you send a screen recording?

**Comment by Ushie** [comment]
![Screenshot_20230804_150610](https://github.com/ReVanced/revanced-manager/assets/79272171/249dda7b-cd09-4ec0-908d-8f8fcc5f6510)
![SmartSelect_20230804_150631](https://github.com/ReVanced/revanced-manager/assets/79272171/90ef5857-333d-40cf-877a-14f7fc668f33)

**Comment by TheAabedKhan** [comment]
Oh, ok! I got confused between experimental patches and universal patches.

You are right! PR on the way.

**Comment by ILoveOpenSourceApplications** [comment]
Can we consider #888 to be solved with this implementation as well?

**Comment by TheAabedKhan** [comment]
@ILoveOpenSourceApplications I will create a new PR for this which will actually separate them. Right now, if universal patches are enabled, they are mixed with the other patches.

**Comment by ILoveOpenSourceApplications** [comment]
Yes, so either we can hide universal patches and only show it to users who enable them in settings or have what the user in #888 say, a separate section for universal patches be implemented. I think one or the other is enough and implementing both would be unnecessary. Any opinion?

**Comment by TheAabedKhan** [comment]
>I think one or the other is enough and implementing both would be unnecessary.

A separate section like [this](https://github.com/ReVanced/revanced-manager/pull/1087#issuecomment-1665654261) can help easily separate the universal patches even when all the patches are shown in the same page.

**Comment by Ushie** [comment]
> Yes, so either we can hide universal patches and only show it to users who enable them in settings or have what the user in #888 say, a separate section for universal patches be implemented. I think one or the other is enough and implementing both would be unnecessary. Any opinion?

Implementing both is definitely necessary:
1. your average consumer has no idea what the difference between universal and package-specific patches is
2. there is no differentiator between universal and package-specific patches, so even ones that know the difference would be unable to specify which is which

**Comment by validcube** [comment]
🎉 _Hooray!_ This issue have been fixed in #1087, please wait for the next version of ReVanced Manager.

Feel free to reopen if you like to another solution.
